props/saber_BC_buttons.h

```diff
@@ -425,6 +1403,34 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
   SaberBCButtons() : PropBase() {}
   const char* name() override { return "SaberBCButtons"; }
 
+#if defined(DYNAMIC_BLADE_LENGTH) && !defined(MENU_SPEC_TEMPLATE)
+  virtual void EnterBladeLengthMode() {
```

Comment:
don't make things virtual if they don't need to be (wastes a little bit of memory)

Reply:
Not sure why that was ever virtual. Fixed.